### DISCUSSION TITLE
Load xodr in UEeditor without fbx

### DISCUSSION THREAD

**Discussion by Sephiroth1118** [carla-simulator/carla#8107] [Q&A]
Hi! 
I'm now working on a project that needs to create a map in Carla which should contains buildings. The standalone mode doesn't meet the requirement and I do not have the license of Roadrunner. What I'm trying to do is load the xodr file inUEeditor as a bare map like in standalone mode and then add some buildings manually. But I found make import do not work for xodr file only. Is there anyway to store the bare map in standalone mode as umap so that I can edit it in UEeditor?
Thank you so much!

**Comment by maybe2122** [comment]
I meet the same issue too.

**Reply by Sephiroth1118** [reply]
Finally I found some way to convert the xodr file into .obj file, which is another format of 3D mesh file. Then .obj can be converted into .fbx. But in this way it can only import the mesh of the road without corresponding relation with the xodr file. In UE editor you need to manually set some parameter to make them align.
